DB/src/main/java/io/deephaven/db/tables/lang/DBLanguageFunctionUtil.java

```diff
@@ -19375,7 +19375,7 @@ public static String doStringPyCast(Object a) {
         if (o == null || o.isNone()) {
             return null;
         }
-        return o.getStringValue();
+        return o.str();
```

Comment:
I'm a little nervous about this change. The previous impl was more like a String cast, this is more like a toString(). Semantically, the former feels more correct for this use case. Maybe @chipkent or @jmao-denver can offer guidance as to whether it's more Pythonic to do a string conversion instead of cast.